docker-images/artifacts/kafka-thirdparty-libs/3.1.x/pom.xml

```diff
@@ -339,12 +339,12 @@
         <dependency>
             <groupId>com.fasterxml.jackson.dataformat</groupId>
             <artifactId>jackson-dataformat-yaml</artifactId>
-            <version>2.12.3</version>
+            <version>2.12.6</version>
         </dependency>
         <dependency>
             <groupId>com.fasterxml.jackson.datatype</groupId>
             <artifactId>jackson-datatype-jsr310</artifactId>
-            <version>2.12.3</version>
+            <version>2.12.6</version>
         </dependency>
```

Comment:
Isn't Kafka 3.1 using 2.12.3? This should stick with the Kafka version to keep things in sync.

Reply:
Kafka has already applied 2.12.6 : 
Kafka 3.0: https://github.com/apache/kafka/commit/ac91fecbd2c1fad7a80a9e9845aae10621756fd7
Kafka 3.1: https://github.com/apache/kafka/commit/f514b9e44649b56e0d436dba9f332b5e7569f93f

Also please refer the ticket at https://issues.apache.org/jira/browse/KAFKA-13658

One option could be to wait until 3.0.1 and 3.1.1 are released, and then include them. 

What is the principle/rules/relation between Strimzi release and Kafka release cycle?

Reply:
Yeah, but these commits mean nothing. They are not in the 3.1.0 or 3.0.0 releases which Strimzi uses.

Reply:
I will change this PR to only have changes related to Cruise Control, so that it by default include Kafka 3.x versions. And remove the explicit Jackson version upgrades.

What would be the process when version 3.0.1 and 3.1.1 of Kafka are released? Will they be included as part of 0.28.1 version of Strimzi, or version 0.29.0?

Reply:
> I will change this PR to only have changes related to Cruise Control, so that it by default includes Kafka 3.x versions. And remove the explicit Jackson version upgrades.

Please do not modify the dependencies randomly just to fix some report. The software is somehow released and tested. The changes we do to it should be only very limited.

Reply:
> What would be the process when version 3.0.1 and 3.1.1 of Kafka are released? Will they be included as part of 0.28.1 version of Strimzi, or version 0.29.0?

They need to be released first. Then they can be included in the next release.